scripts/release/steps/update-changelog.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+const chalk = require("chalk");
+const dedent = require("dedent");
+const fs = require("fs");
+const prsMergedSince = require("prs-merged-since");
+const semver = require("semver");
+const { logPromise, keypress } = require("../utils");
+
+async function prepareReleaseNotes({ version, previousVersion }) {
+  const versionDiff = semver.diff(version, previousVersion);
+
+  let changes;
+  let warning;
+  let releaseNotes;
+
+  if (versionDiff !== "patch") {
+    // Changelog is the blog post, so we just add a link to it
+    changes = "blog post link";
+
+    const date = new Date();
+    const year = date.getFullYear();
+    const month = new String(date.getMonth() + 1).padStart(2, "0");
+    const day = new String(date.getDate()).padStart(2, "0");
+    const blogPostFile = `website/blog/${year}-${month}-${day}-${version}.md`;
+    const blogPostPath = `${year}/${month}/${day}/${version}.html`;
+
+    releaseNotes = `- [Release Notes](https://prettier.io/blog/${blogPostPath})`;
+
+    if (!fs.existsSync(blogPostFile)) {
+      warning = chalk`Make sure the blog post file {yellow ${blogPostFile}} will be created.`;
```

Comment:
This message is confusing me- is the user supposed to create a blog post before running the script? Or will the script create a skeleton that the user is supposed to edit before continuing?

Reply:
@suchipi Neither. The user just needs to make sure this file exists or will exist in the future, just so the link in CHANGELOG works.

How can I make it more clear?

I also added this because of the last blog post in which I committed with the wrong filename.

Reply:
We might want to die here instead and ask the user to create the file before running the script, because a blog post should be required for non-patch releases.

Reply:
@suchipi I don't understand, why should the script die here?

I don't think the blog post should block the release. We'll either merge the blog post before or after editing `CHANGELOG.md` anyway. So the two scenarios are:

- post does not exist because PR is still open
  - runs the script
  - updates `CHANGELOG.md` and prints a warning for what the blog post path should be
  - publish to npm
  - verify blog post path in PR
  - merge blog post PR
  - tweet

- post does not exist because path is wrong
  - runs the script
  - updates `CHANGELOG.md` and prints a warning for what the blog post path should be
  - publish to npm
  - fix blog post path
  - commit
  - tweet

- post exists
  - runs the script
  - updates `CHANGELOG.md`
  - publish to npm
  - tweet

Also I initially want this script to run completely at once so we don't have to remember the order of the "script commands" we need to run. Then we can add a few `--skip-X` if we see an use case where it's useful

Reply:
Alright, I understand after reading through your examples. I agree then, it shouldn't die. Let's just change the message to:

```